tests/providers/amazon/aws/hooks/test_batch_waiters.py

```diff
@@ -85,15 +85,15 @@ def test_default_config(self):
         assert config["version"] == 2
         assert isinstance(config["waiters"], dict)
 
-        waiters = list(sorted(config["waiters"].keys()))
+        waiters = sorted(config["waiters"].keys())
```

Comment:
```suggestion
        waiters = sorted(config["waiters"])
```

Reply:
There are lots of unneeded `.keys()` in the code. Will add a new commit.